airflow/providers/microsoft/azure/provider.yaml

```
@@ -97,6 +99,9 @@ operators:
   - integration-name: Microsoft Azure Blob Storage
     python-modules:
       - airflow.providers.microsoft.azure.operators.wasb_delete_blob
+  - integration-name: Microsoft Azure Data Factory
+    python-modules:
+      - airflow.providers.microsoft.azure.operators.azure_data_factory
```

```suggestion
      - airflow.providers.microsoft.azure.operators.data_factory
```
see sensor path comment